tools/epicseg/epicseg_segment.xml

```diff
@@ -0,0 +1,127 @@
+<tool id="epicseg_segment" name="EpiCSeg - Segment" version="0.1.0">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="1.0">epicseg</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '${__tool_directory__}/epicseg.R' segment --regions "$regions"
+           --nstates $nstates
+           --nthreads $nthreads
+
+        #for $dataset in $datasets#
+           --counts ${dataset.name}:${dataset.count_matrix}
+        #end for#
+
+           --maxiter $maxiter
+
+        #for $annot in $annotation#
+           --annot ${annot.name}:${annot.annotation}
+        #end for# &&
+
+        mkdir -p '${ report.files_path }' &&
+        cp lmeans.txt '${ report.files_path }' &&
```

Comment:
you can use `from_work_dir` like here: https://github.com/galaxyproject/tools-iuc/blob/6236c2beed98e11da74605822fc27bd222354816/tools/roary/roary.xml#L74

This way you don't need to copy data in your tool.